operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -228,15 +268,15 @@ func (r *TenantReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	}
 
 	// no retrigErr, need to normal reconcile later, so need to create random number and exit
-	nextRequeSeconds, err := randomRange(3600*4, 3600*8) // need to use seconds value for interval 4h-8h to have resolution to the second
+	nextRequeueSeconds, err := randomRange(int(r.RequeueTimeMinimum.Seconds()), int(r.RequeueTimeMaximum.Seconds())) // need to use seconds value for interval 4h-8h to have resolution to the second
```

Comment:
The comment shall be removed, as no longer true.

Reply:
Resolved in [f95748a](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/f95748a2341ca4e93791b8f28f433fb5a828b3f9)